src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -436,15 +436,28 @@ bool AletheProofPostprocessCallback::update(Node res,
     case ProofRule::CHAIN_RESOLUTION:
     {
       std::vector<Node> newArgs;
+      std::vector<Node> cargs;
+      if (id == ProofRule::CHAIN_RESOLUTION)
```

Comment:
this test does not make sense since this is within a case where id is chain_resolution

Reply:
There are 2 cases RESOLUTION and CHAIN_RESOLUTION handled here.

Reply:
sorry, github fooled me :)